### PR TITLE
Improvements to the helper scripts

### PR DESCRIPTION
**Pull request by dumbbell**
Here are a few improvements to the scripts we use to extract and apply patches from Linux to FreeBSD.

### CODE REVIEW

File: scripts/drmgeneratepatch [hunk 1]
```diff
@@ -14,9 +14,9 @@ if [ $# -ne 3 ]; then
 	usage
 fi
 
-LINUX_DIR=$1
+LINUX_DIR=$(cd "$1" && pwd)
```
Comment (evadot) [inline scripts/drmgeneratepatch:17]: realpath might be better.

File: scripts/drmgeneratepatch [hunk 2]
```diff
@@ -14,9 +14,9 @@ if [ $# -ne 3 ]; then
...
 shift
-PATCH_DIR=$1
+PATCH_DIR=$(cd "$1" && pwd)
```
Comment (evadot) [inline scripts/drmgeneratepatch:19]: same comment for real path.

**Review by evadot — CHANGES_REQUESTED** [review]

File: scripts/drmgeneratepatch (hunk 1 above)
Comment (dumbbell) [reply scripts/drmgeneratepatch:17]: Probably :-) I'm used to use cd(1) because realpath(1) is not available everywhere (or at least wasn't).

Comment (dumbbell) [reply scripts/drmgeneratepatch]: Fixed.

**Pull request merged**